loky/backend/context.py

```diff
@@ -67,6 +69,43 @@ def get_context(method="loky"):
                              "methods are {}".format(method, METHODS))
 
 
+def cpu_count():
+    """Return the number of CPUs the current process can use.
+    """
+    import math
+
+    try:
+        cpu_count_mp = mp.cpu_count()
+    except NotImplementedError:
+        cpu_count_mp = 1
+
+    # Number of available CPUs given affinity settings
+    cpu_count_affinity = cpu_count_mp
+    if hasattr(os, 'sched_getaffinity'):
+        try:
+            cpu_count_affinity = len(os.sched_getaffinity(0))
+        except NotImplementedError:
+            pass
+
+    # CFS scheduler CPU bandwidth limit
+    # available in Linux since 2.6 kernel
+    cpu_count_cfs = cpu_count_mp
+    cfs_quota_fname = "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"
+    cfs_period_fname = "/sys/fs/cgroup/cpu/cpu.cfs_period_us"
+    if (os.name == 'posix' and os.path.exists(cfs_quota_fname)
+            and os.path.exists(cfs_period_fname)):
+        with open(cfs_quota_fname, 'r') as fh:
+            cfs_quota_us = int(fh.read())
+        with open(cfs_period_fname, 'r') as fh:
+            cfs_period_us = int(fh.read())
+
+        if cfs_quota_us > 0:
+            cpu_count_cfs = math.ceil(cfs_quota_us / cfs_period_us)
+            cpu_count_cfs = max(cpu_count_cfs, 1)
+
+    return min(cpu_count_mp, cpu_count_affinity, cpu_count_cfs)
+
+
 class LokyContext(BaseContext):
     """Context relying on the LokyProcess."""
     _name = 'loky'
```

Comment:
Actually `LokyContext.cpu_count` should be overridden to use this new function.

Reply:
Done.